### PR TITLE
[feat]: Added Wishlist Page.

### PR DESCRIPTION
**Pull request by raghav2110**
Hi , I have added wishlist page. Please review PR of same. In order to view wishlist page click on the Wishlist Icon(Heart Icon) on header.

### CODE REVIEW

File: css/wishList.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    justify-content: space-between;
+    padding: 5px;
+}
+
+.btn-addToCart{
+    margin: 0.625rem;
+    background-color: orange;
```
Comment (Hamdrive) [inline css/wishList.css]: Suggestion to consider defining this color as a custom color in a variable instead
As each browser might have their own color code definition for `orange`

File: css/wishList.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    padding: 0.5rem 1rem;
+    font-weight: 800;
+    color: white;
+    border: transparent;
+    border-radius: 0.5rem;
+}
+
+.card-dismiss i{
+    color: red;
```
Comment (Hamdrive) [inline css/wishList.css]: Same as previous comment

File: functionality/remove-from-wishlist.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+function showCodeForComp(id) {
```
Comment (Hamdrive) [inline functionality/remove-from-wishlist.js:1]: JS implemented does not seem to be functional
also might not be required for the current deadline

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Code looks great, good practices followed
Have given suggestions regarding color variables,
and also seems the JS is not working

File: css/wishList.css (hunk 1 above)
Comment (raghav2110) [reply css/wishList.css]: okay noted.

File: functionality/remove-from-wishlist.js (hunk 3 above)
Comment (raghav2110) [reply functionality/remove-from-wishlist.js:1]: It is working ... it is just that all cards are same so the cards seems to be deleting from last. Anyway JS not mandatory for now so keeping that for later.

Comment (Hamdrive) [reply functionality/remove-from-wishlist.js:1]: Noted, did not see the bottom  most row. Will take back that comment, works great!

**Review by Hamdrive — COMMENTED** [review]
Changes requested have been implemented and looks code!
Can be merged, good to go!

File: css/wishList.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    justify-content: space-between;
+    padding: 5px;
+}
+
+.btn-addToCart{
+    margin: 0.625rem;
+    background-color: #FFA500;
+    padding: 0.5rem 1rem;
+    font-weight: 800;
+    color: white;
```
Comment (ZuberDunge) [inline css/wishList.css:39]: you can use css variable s here

File: css/wishList.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    border: transparent;
+    border-radius: 0.5rem;
+}
+
+.card-dismiss i{
+    color: #FF0000;
```
Comment (ZuberDunge) [inline css/wishList.css:45]: same here

**Review by ZuberDunge — APPROVED** [review]
Code looks clean and readable. good to merge.

**Pull request merged**